ui-showdetails/src/main/java/app/tivi/showdetails/details/ShowDetails.kt

```diff
@@ -441,23 +433,46 @@ private fun OverlaidStatusBarAppBar(
     LogCompositions("OverlaidStatusBarAppBar")
 
     Column(modifier) {
-        val props = transition(
-            definition = statusBarTransitionDef,
-            initState = false,
-            toState = showAppBar
+        val transition = updateTransition(showAppBar)
+
+        val elevation by transition.animateDp { value ->
```

Comment:
Name the param?

Reply:
We then get into shadowed names. Not sure what is better here tbh.